hoomd/pytest/test_simulation.py

```diff
@@ -0,0 +1,158 @@
+import hoomd
+import numpy as np
+import pytest
+from copy import deepcopy
+import gsd.hoomd
```

Comment:
We need to catch an import error here and skip the gsd test if the module is not available.

Reply:
Fixed fe9754f7e